docs/CircleCI.md

```diff
@@ -1,72 +1,82 @@
-- `parallelism`: A `parallelism` parameter (if defined greater than 1) will create a seperate execution environment and will run the `steps` of the specific `job` in parallel. In Buildkite - a similar `parallelism` key can be set to a [command step](https://buildkite.com/docs/tutorials/parallel-builds#parallel-jobs) which will run the defined `command` over seperate jobs (sharing the same agent [queues](https://buildkite.com/docs/agent/v3/queues#setting-an-agents-queue)/[tags](https://buildkite.com/docs/agent/v3/cli-start#setting-tags) targets).
-- `shell`: The `shell` property sets the default shell that is used across all commands within all steps. This should be configured on the agent - or by defining the `shell` [option](https://buildkite.com/docs/agent/v3/cli-start#shell) when starting a Buildkite agent: which will set the shell command used to interpret all build commands.
-
-## Unsupported properties (No direct translation)
-
-### orbs
-
-A `orb` within CircleCI is a reusable piece of configuration that can be utilised within a CircleCI pipeline - and often used for third party integration and tasks associated with the services capability.
-
-Orbs are currently not supported by the Buildkite Migration tool, and should be translated by hand if required to utilise within a Buildkite pipeline. The Buildkite platform has reusable [plugins](https://buildkite.com/docs/plugins/directory) that provide a similar experience for integrating various common (and third party integration) tasks throughout a Buildkite pipeline, such as [logging into ECR](https://github.com/buildkite-plugins/ecr-buildkite-plugin), running a step within a [Docker container](https://github.com/buildkite-plugins/docker-buildkite-plugin), running multiple Docker images through a [compose file](https://github.com/buildkite-plugins/docker-compose-buildkite-plugin), triggering builds in a [monorepo setup](https://github.com/buildkite-plugins/monorepo-diff-buildkite-plugin) and many more.
-
-Additionally, an approach to assist in configuring reusable pieces of configuration within a Buildkite pipeline is through the use of [YAML anchors](https://buildkite.com/docs/plugins/using#using-yaml-anchors-with-plugins) - which are defined and used in a similar way to CircleCI `aliases`. 
\ No newline at end of file
+The Buildkite Migration tool's currently supported (🟢), partially supported (🟡) and unsupported (🔴) properties in translation of CircleCI pipelines to Buildkite pipelines are listed below.
+
+### Logical Operators/Helper Keys
+
+| Key | Supported? | Notes |
+| --- | --- | --- |
+| `aliases` | 🟢 | A list of reusable YAML snippets of a CircleCI pipeline.
+| `aliases.&<name>` | 🟢 | A singular alias describing a resusable snippet of YAML to apply to a specific point in a CircleCI pipeline. Defined with a `&` (anchor) - these blocks are substituted into configuration with `*`: for example, `*tests`.
```

Comment:
I'm not sure these two rows should be in the document. In this section for sure not as they are not actual keys nor CircleCI-specific, just part of the YAML standard.

Maybe just add a note that anchors and aliases will be respected and parsed correctly.

Reply:
Agreed - I've split aliases/anchors which now lives in a note 👍